app/src/test/java/org/simple/clinic/summary/addphone/AddPhoneNumberDialogControllerTest.kt

```diff
@@ -118,11 +112,14 @@ class AddPhoneNumberDialogControllerTest {
   @Suppress("unused")
   private fun `validation errors`() = listOf(Blank, LengthTooShort(6), LengthTooLong(12))
 
-  private fun setupController() {
+  private fun setupController(newNumber: String) {
     val controller = AddPhoneNumberDialogController(repository, validator, FakeUuidGenerator.fixed(generatedPhoneUuid))
 
     controllerSubscription = uiEvents
         .compose(controller)
         .subscribe { uiChange -> uiChange(dialog) }
+
+    uiEvents.onNext(AddPhoneNumberDialogCreated(patientUuid))
+    uiEvents.onNext(AddPhoneNumberSaveClicked(newNumber))
```

Comment:
Can we have this particular event done separately in each test? We should only move emission of the `ScreenCreated` event since that is a part of screen initialization and something that is supposed to be triggered for every test.

Reply:
You're talking about `AddPhoneNumberSaveClicked` event right?

Reply:
Yes.